src/main/java/catascore/KeyGeneratorImpl.java

```diff
@@ -1,22 +1,26 @@
 package catascore;
 
-public class KeyGeneratorImpl implements KeyGenerator {
+
+public class KeyGeneratorImpl  implements KeyGenerator{
 
     private int first = 0;
```

Comment:
It would be better to change int to boolean, it is not understood what this field is

Reply:
Changed, thanks